crates/revm/src/db.rs

```diff
@@ -17,14 +17,14 @@ use auto_impl::auto_impl;
 #[auto_impl(& mut, Box)]
 pub trait Database {
     /// Get basic account information.
-    fn basic(&mut self, address: H160) -> AccountInfo;
+    fn basic(&mut self, address: H160) -> Result<Option<AccountInfo>, &'static str>;
```

Comment:
also I think it'd be useful to use an error alias and reexport it

something like:
```rust
pub type DbResult<T> = Result<T, &'static str>;

```

Reply:
`Error` type fixes this